crates/bdk/src/wallet/mod.rs

```diff
@@ -109,6 +110,38 @@ pub struct Update {
     pub chain: Option<local_chain::Update>,
 }
 
+impl
+    From<(
+        BTreeMap<KeychainKind, u32>,
+        TxGraph<ConfirmationTimeHeightAnchor>,
+        local_chain::Update,
+    )> for Update
+{
+    fn from(
+        (last_active_indices, graph, chain): (
+            BTreeMap<KeychainKind, u32>,
+            TxGraph<ConfirmationTimeHeightAnchor>,
+            local_chain::Update,
+        ),
+    ) -> Self {
+        Self {
+            last_active_indices,
+            graph,
+            chain: Some(chain),
+        }
+    }
+}
+
+impl From<(TxGraph<ConfirmationTimeHeightAnchor>, local_chain::Update)> for Update {
+    fn from((graph, chain): (TxGraph<ConfirmationTimeHeightAnchor>, local_chain::Update)) -> Self {
+        Self {
+            graph,
+            chain: Some(chain),
+            ..Default::default()
+        }
+    }
+}
```

Comment:
Are these `From` impls really important. These kinds of things are hard for users to find even if they are useful. It's better not to have a codebase full of secret tricks.

Reply:
Will remove.